tests/system/query_generator/test_cohere_query_generator.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+
+from canopy.chat_engine.query_generator.cohere import CohereQueryGenerator
+from canopy.models.data_models import MessageBase, Role
+
+
+@pytest.fixture
+def messages():
```

Comment:
@igiloh-pinecone I changed it to use the fixture from `conftest.py` in the other place you mentioned it, but I ran into an issue with doing it here. The problem is that the last message from the fixture is "Just checking in. Be concise.", but because this message does not have a question from the user, the Cohere Chat API with the `generate_search_queries=True` parameter does not return any recommended search queries.

I guess I could use that fixture and then append a real question to the messages in the test.

Reply:
If it's only one test case - I guess appending a question inside the test func sounds right.  
Up to you.